### ISSUE TITLE
ConfigFile allows spaces but removes them on load

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#18664]
Godot Master
x11

Steps:
1. Run the code
```
extends Node

var config = ConfigFile.new()
var config_path = "user://settings.cfg"


func _ready():
	
	
	if config.load(config_path) != OK:
		config.save(config_path)
	
	config.set_value("global","Test Test",false) # Comment this in step 2
	config.save(config_path) # Comment this in step 2
	
	print(config.get_section_keys("global"))
```
Outout: [Test Test]
settings.cfg
```
[global]

Test Test=false
```

2. Comment the 2 lines and run again
Output: [TestTest]

3. Add the 2 lines back in and run
Output: [TestTest, Test Test]

**Comment by reduz** [comment]
This is because the variant parser allows parameters in the tags, but ConfigFile does not use them.
I don't think this is a bug, it's just extra information that gets ignored when you save, just like comments.

Will document this and close it

**Comment by reduz** [comment]
Closed by 35b421b69.
